plugins/hls-eval-plugin/test/testdata/TMultiResult.expected.hs

```diff
@@ -0,0 +1,13 @@
+module TMultiResult where
+-- test multiline show instance (see #2907)
+
+data Multiline = M {l1 :: String, l2 :: String} deriving Read
+
+instance Show Multiline where
+  show m = "M {\n  l1=" <> show (l1 m) <> ",\n  l2=" <> show (l2 m) <> "\n}"
+
+-- >>> M "first line" "second line"
+-- M {
+--   l1="first line",
+--   l2="second line"
+-- }
```

Comment:
This only tests `--` comments, but I figured `{- -}` would be fine and covered by the other multiline tests.